gcloud-java-dns/src/main/java/com/google/gcloud/dns/ZoneInfo.java

```diff
@@ -66,19 +66,10 @@
     private Builder() {
```

Comment:
Correct if we assume that the pb object always has initialized name. This assumption should be safe to make with the exception of specifying giving the user the option to specify fields of zone to be returned. I rectified the issue by enforcing the field selectors of zone to always return zone name instead of zone id which it was set to up to now. Removing the builder.